syntax = "proto2";

option java_multiple_files = true;
option java_outer_classname = "ProtoLongArrayWrapper";

package org.openstreetmap.atlas.proto;

message ProtoLongArray {
    optional string name = 1;
    repeated int64 elements = 2;
}
